src/app/gateways/components/RuleFilter.vue

```diff
@@ -0,0 +1,145 @@
+                <KBadge>{{ t(`http.api.property.${key}`) }}</KBadge>
+
+                <span
+                  v-for="(entry, index) in value"
+                  :key="index"
+                >
+                  <template v-if="typeof entry === 'string'">
+                    {{ entry }}
+                  </template>
+
+                  <template v-else>
+                    {{ entry.name }}:{{ entry.value }}
+                  </template>
+                </span>
+              </div>
+            </template>
+          </template>
+        </div>
+      </template>
+
+      <template v-else-if="props.filter.type === 'RequestMirror'">
+        <TargetRef :target-ref="props.filter.requestMirror.backendRef">
+          {{ props.filter.requestMirror.backendRef.name }}
+          <template v-if="props.filter.requestMirror.percentage">
+            ({{ props.filter.requestMirror.percentage }})
+          </template>
+        </TargetRef>
+      </template>
+
+      <template v-else-if="props.filter.type === 'RequestRedirect'">
+        <div class="list">
+          <div
+            v-for="(value, key) in props.filter.requestRedirect"
+            :key="key"
+          >
+            <template v-if="value">
+              <span class="text-neutral">
+                {{ t(`http.api.property.${key}`) }}:
+              </span>
+
+              <template v-if="typeof value === 'object'">
+                {{ value.type === 'ReplaceFullPath' ? value.replaceFullPath : value.replacePrefixMatch }}
+              </template>
+
+              <template v-else>
+                {{ value }}
+              </template>
+            </template>
+          </div>
+        </div>
+      </template>
+
+      <template v-else>
+        <div class="list">
+          <div
+            v-for="(value, key) in props.filter.urlRewrite"
+            :key="key"
+          >
+            <template v-if="value">
+              <span class="text-neutral">
+                {{ t(`http.api.property.${key}`) }}:
+              </span>
+
+              <template v-if="typeof value === 'object'">
+                {{ value.type === 'ReplaceFullPath' ? value.replaceFullPath : value.replacePrefixMatch }}
+              </template>
+
+              <template v-else>
+                {{ value }}
+              </template>
+            </template>
+          </div>
+        </div>
+      </template>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import TargetRef from '@/app/common/TargetRef.vue'
+import type { ToTargetRefFilter } from '@/types/index.d'
+import { useI18n } from '@/utilities'
+
+const { t } = useI18n()
+
+const props = defineProps<{
+  filter: ToTargetRefFilter
+}>()
+</script>
+
+<style lang="scss" scoped>
+.filter {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $kui-space-50;
+}
+
+.list {
+  display: inline-flex;
+  flex-wrap: wrap;
+  gap: $kui-space-40;
+  align-items: baseline;
+}
+
+.text-neutral {
+  color: $kui-color-text-neutral;
+}
+</style>
```

Comment:
Would it make sense for this and RuleMatch go into the `rules` module?

Reply:
Yes, I’m moving them provided the directory already exists.

Reply:
Done.